app/graphql/types/debate_type.rb

```diff
@@ -2,17 +2,18 @@ module Types
   class DebateType < Types::BaseObject
     field :id, ID, null: false
     field :title, String, null: true
-    field :description, String, null: true
-    field :public_created_at, String, null: true
-    field :cached_votes_total, Integer, null: true
-    field :cached_votes_up, Integer, null: true
-    field :cached_votes_down, Integer, null: true
-    field :comments_count, Integer, null: true
-    field :hot_score, Integer, null: true
-    field :confidence_score, Integer, null: true
     field :comments, Types::CommentType.connection_type, null: true
-    field :public_author, Types::UserType, null: true
-    field :votes_for, Types::VoteType.connection_type, null: true
+    field :description, String, null: true
+
+    field_with_deprecated_sibling :public_created_at, String, null: true
+    field_with_deprecated_sibling :cached_votes_total, Integer, null: true
+    field_with_deprecated_sibling :cached_votes_up, Integer, null: true
+    field_with_deprecated_sibling :cached_votes_down, Integer, null: true
+    field_with_deprecated_sibling :comments_count, Integer, null: true
+    field_with_deprecated_sibling :hot_score, Integer, null: true
+    field_with_deprecated_sibling :confidence_score, Integer, null: true
+    field_with_deprecated_sibling :public_author, Types::UserType, null: true
+    field_with_deprecated_sibling :votes_for, Types::VoteType.connection_type, null: true
```

Comment:
maybe putting this block below `field :tags,` would result in a better readability with quicker overview of fields and fields with deprecated siblings?